### DISCUSSION TITLE
Where can i see my day benefit (sum benefit of every product }

### DISCUSSION THREAD

**Discussion by killuadk** [Blair2004/NexoPOS#1213] [Ideas]
Hi guys
First of all Nice work was done 👍
As title i couldn't find where can i see my benefit of the day 

For example 
I did sell 3xProduct X that i buy 3k and i did sell it 3k5 
I want to see that i made 1k5 profits (500x3)

**Comment by killuadk** [comment]
Hi gyus 
After looking at this file "resources\views\pages\dashboard\reports\profit-report.blade.php" i added  **product.total_purchase_price*product.quantity** so it look like 
`        <td class="p-2 border text-right border-info-primary">@{{ ( product.total_price - product.total_purchase_price*product.quantity ) | currency }}</td>`
i was able to see my profit of the day But i face another problem that product.total_price doesn't take in consideration the Discount  
example i sale product A 1000  for friend instead of 1200 
in profit section it show 1200

**Comment by Blair2004** [comment]
Hey, 
Sorry the discussion went unnoticed. We'll consider this, but you should make a valid pull request so we can consider a possible merge.
